data/colors_v2/themes/dark_high_contrast.ts

```diff
@@ -190,6 +190,9 @@ const exceptions = {
     expander: {
       icon: get('fg.onEmphasis'),
     },
+  },
+  tooltip: {
+    bg: get('scale.gray.1')
```

Comment:
If we need to add an exception for `tooltip.bg`, we'll have to undeprecate it by moving it back to the `vars/components_*.ts` files.

Reply:
What does `neutral.emphasisPlus` look like for tooltips in high contrast mode?

Reply:
Tried to simulate it:

`scale.gray.1` | `neutral.emphasisPlus`
--- | ---
<img width="398" alt="Screen Shot 2021-07-21 at 10 54 03" src="https://user-images.githubusercontent.com/378023/126418425-f5cbd8ed-00f9-4b5e-8712-cc3ad6f8726d.png"> | <img width="365" alt="Screen Shot 2021-07-21 at 10 52 57" src="https://user-images.githubusercontent.com/378023/126418413-9d54a2c1-c9ac-4032-bd74-fcb14f85f5cc.png"> 

It's not too contrasty, but I wonder if `neutral.emphasisPlus` should be `white` or `gray-0` in high contrast? I think we don't really use `neutral.emphasisPlus` for anything yet, so might be ok to still change it?

Reply:
@simurai agree `gray-0` makes more sense here for `emphasisPlus`